Dockerfile

```diff
@@ -82,44 +154,35 @@ RUN mkdir -p /etc/sysconfig/slurm \
 
 RUN ssh-keygen -q -t rsa -N '' -f /root/.ssh/container_root_pubkey
 
-# setup login to ssh
-RUN mkdir -p /root/.ssh/ \
-    && cat /root/.ssh/container_root_pubkey.pub >> /root/.ssh/authorized_keys \
-    && chmod -R 700 /root/.ssh/ \
-    && sed -i 's/#PasswordAuthentication yes/PasswordAuthentication yes/g' /etc/ssh/sshd_config \
-    && echo 'Port=2222' >> /etc/ssh/sshd_config \
-    && echo 'PermitRootLogin yes' >> /etc/ssh/sshd_config
+# Set up login to SSH
+RUN mkdir -p /root/.ssh/ && \
+    cat /root/.ssh/container_root_pubkey.pub >> /root/.ssh/authorized_keys && \
+    chmod -R 700 /root/.ssh/ && \
+    sed -i 's/#PasswordAuthentication yes/PasswordAuthentication yes/g' /etc/ssh/sshd_config && \
+    echo 'Port=2222' >> /etc/ssh/sshd_config && \
+    echo 'PermitRootLogin yes' >> /etc/ssh/sshd_config
 
-# ssh setup https://askubuntu.com/questions/1110828/ssh-failed-to-start-missing-privilege-separation-directory-var-run-sshd
-RUN mkdir -p /var/run/sshd \
-    && chmod 0755 /var/run/sshd
+# SSH set up
+# Ref: https://askubuntu.com/questions/1110828/ssh-failed-to-start-missing-privilege-separation-directory-var-run-sshd
+RUN mkdir -p /var/run/sshd && \
+    chmod 0755 /var/run/sshd
 
 # Create folder to simulate the projects used in Marenostrum and for the tests
 RUN mkdir -p /tmp/scratch/group/root
 
-# Set the locale. Taken from http://jaredmarkell.com/docker-and-locales/
-
-RUN sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen \
-    && locale-gen
-
-ENV LANG=en_US.UTF-8
-ENV LANGUAGE=en_US:en
-ENV LC_ALL=en_US.UTF-8
-
-# Add Tini. (manuel: a lightweight init, systemd, etc)
-
-ENV TINI_VERSION=v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
-RUN chmod +x /tini
+# Set the locale.
+# Ref: https://jaredmarkell.com/docker-and-locales/
+RUN sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen && \
+    locale-gen
 
 # Copy Slurm configuration files into the container
 COPY slurm.conf /etc/slurm/slurm.conf
 COPY slurmdbd.conf /etc/slurm/slurmdbd.conf
+COPY cgroup.conf /etc/slurm/cgroup.conf
 COPY docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
 
 RUN chmod 600 /etc/slurm/slurm.conf /etc/slurm/slurmdbd.conf
 
 EXPOSE 2222
 
-CMD ["/tini", "--", "/usr/local/bin/docker-entrypoint.sh"]
-
+ENTRYPOINT ["/usr/bin/tini", "--", "/usr/local/bin/docker-entrypoint.sh"]
```

Comment:
Made the entrypoint an entrypoint.

If you want to execute something with the container instead of the entrypoint, just `docker run -ti --entrypoint "" .... /bin/bash` :+1: